main.py

```diff
@@ -1,17 +1,121 @@
 #!/usr/bin/env python3
 ############################################################################
 # Soubor:  main.py
-# Datum:
-# Autor:
+# Datum: 10.1.2022
+# Autor: Michal Vojtěchovský
 ############################################################################
-from random import randint, choice
+from random import randint, choice, random
+import time
 
 ############################################################################
 
+def lowerCaseConvertor(file):
+    try:
+        f = open(f"{file}.txt","r")
+    except FileNotFoundError:
+        print("soubor nenalezen")
+        menu()    
+    f2 = open(f"{file}_lowerCase.txt","w")
+    for text in f:
+        f2.write(text.lower())
+    print(f"převod dokončen do souboru {file}_lowerCase.txt")    
+    f.close()  
+    f2.close()
+    return f2
 
-def f():
-    return None
 
+def letterChanger(file):
+    try:
+        f = open(f"{file}.txt","r")
 
-if __name__ == "__main__":
-    print(f())
+    except FileNotFoundError:
+        print("soubor nenalezen")
+        menu()        
+    letter1 = input("Zadej písmeno, kterým chceš nahrazovat: ")
+    letter2 = input("Zadej písmeno, které cheš nahradit: ")            
+    f2 = open(f"{file}_changedLetters.txt","w")
+    while True:
+        char = f.read(1)   
```

Comment:
Když čtete 1B, tak vám nebude to nebude fungovat pro ne-ascii znaky u vícebytových znakových sad jako je např UTF-8